package.json

```diff
@@ -98,10 +99,9 @@
   },
   "lint-staged": {
     "*.js": [
-      "prettier --write",
+      "prettier --write --trailing-comma es5 --print-width 120 --no-semi",
```

Comment:
VS Code uses an extension (it's in the recommendations), that's set up with the same settings